obstacledetection/src/main/java/com/caneseeproject/obstacledetection/ObstacleDetectorImpl.kt

```diff
@@ -42,7 +42,7 @@ internal class ObstacleDetectorImpl(private val odPortal: SensorPortal) : Obstac
      */
     private fun odInputEncoder(highLevelInput: ODInput): String {
         return when (highLevelInput) {
-            is ODInput.RangeControl -> "${highLevelInput.percentage}"
+            is ODInput.RangeControl -> "2_${highLevelInput.percentage}"
```

Comment:
Just asking, why starting from “2”?

Reply:
Because number "0" is assigned to the `ODReading.ObstacleDistance(distance)` , also number "1" with the glasses mode , so why repeating numbers that may confuse myself in the future ?!
And what if in the future and somehow one of these types will pass from and into the cane ?

Reply:
Then, I think that i have to make every "type" number in the `type_value` a Global variable.

Reply:
> Because number "0" is assigned to the `ODReading.ObstacleDistance(distance)` , also number "1" with the glasses mode , so why repeating numbers that may confuse myself in the future ?!
> And what if in the future and somehow one of these types will pass from and into the cane ?

But that's for a completely different thing, isn't it? starting from two is actually confusing is it might convey that it's in the same standard as the readings, even though it clearly isn't.

> Then, I think that i have to make every "type" number in the type_value a Global variable.
Yes, that's really important.

Reply:
> is it might convey that it's in the same standard as the readings, even though it clearly isn't.

Didin't get what do you mean ?

Reply:
We can discus this face to face tomorrow.